scripts/build/p-sqlite.inc

```diff
@@ -24,7 +24,7 @@ build_sqlite() {
     [[ -z "${COMP}" ]] && COMP="$(which gcc)"
   fi
 
-  ${COMP} ${CFLAGS} -fPIC -o libsqlite3.so -DSQLITE_THREADSAFE=0 -DSQLITE_OMIT_LOAD_EXTENSION -shared sqlite3.c
+  ${COMP} ${CFLAGS:-} -fPIC -o libsqlite3.so -DSQLITE_THREADSAFE=0 -DSQLITE_OMIT_LOAD_EXTENSION -shared sqlite3.c
```

Comment:
Can you change it to the following:
```suggestion
  "${COMP}" "${CFLAGS[@]}" -fPIC -o libsqlite3.so -DSQLITE_THREADSAFE=0 -DSQLITE_OMIT_LOAD_EXTENSION -shared sqlite3.c
```

Reply:
I changed it according to your suggestion.

Reply:
Update: I also had to change `CFLAGS` a few lines above, as it was defined as ```CFLAGS="${SANITIZER_C_FLAGS[*]}"```, a flat string. Now, I'm just making a copy of the array.